frontend/common/stores/account-store.js

```diff
@@ -85,24 +89,29 @@ const controller = {
       API.postEvent(`${name}`)
     }
 
-    data.post(`${Project.api}organisations/`, { name }).then((res) => {
-      store.model.organisations = store.model.organisations.concat([
-        { ...res, role: 'ADMIN' },
-      ])
-      AsyncStorage.setItem('user', JSON.stringify(store.model))
-      store.savedId = res.id
-      store.saved()
+    data
+      .post(`${Project.api}organisations/`, {
+        hubspotutk: API.getCookie('hubspotutk'),
+        name,
+      })
+      .then((res) => {
+        store.model.organisations = store.model.organisations.concat([
+          { ...res, role: 'ADMIN' },
+        ])
+        AsyncStorage.setItem('user', JSON.stringify(store.model))
+        store.savedId = res.id
+        store.saved()
 
-      const relayEventKey = Utils.getFlagsmithValue('relay_events_key')
-      const sendRelayEvent =
-        Utils.getFlagsmithHasFeature('relay_events_key') && !!relayEventKey
-      window.lintrk?.('track', { conversion_id: 16798338 })
-      if (sendRelayEvent) {
-        dataRelay.sendEvent(AccountStore.getUser(), {
-          apiKey: relayEventKey,
-        })
-      }
-    })
+        const relayEventKey = Utils.getFlagsmithValue('relay_events_key')
+        const sendRelayEvent =
+          Utils.getFlagsmithHasFeature('relay_events_key') && !!relayEventKey
+        window.lintrk?.('track', { conversion_id: 16798338 })
```

Comment:
What is `conversion_id` here? If it's a magic number we should assign it to a meaningful variable name like `google_account_tracking_id = 16798338` before using it in the track action.

Reply:
Yep good point, done